src/main/java/com/starkinfra/IssuingStock.java

```diff
@@ -0,0 +1,582 @@
+         * @param stock [IssuingStock]: IssuingStock entity to which the log refers to.
+         * @param type [string]: type of the IssuingStock event which triggered the log creation. ex: "created", "spent", "restocked", "lost"
+         * @param count [integer]: shift in stock balance. ex: 10
+         * @param created [string]: creation datetime for the log. ex: "2020-03-10 10:30:00.000000+00:00"
+         */
+        public Log(String id, IssuingStock stock, String type, Integer count, String created ) {
+            super(id);
+            this.stock = stock;
+            this.type = type;
+            this.count = count;
+            this.created = created;
+        }
+
+        /**
+         * Retrieve a specific IssuingStock Log
+         * <p>
+         * Receive a single IssuingStock Log object previously created by the Stark Infra API by passing its id
+         * <p>
+         * Parameters:
+         * @param id [string]: object unique id. ex: "5656565656565656"
+         * <p>
+         * Return:
+         * @return IssuingStock Log object with updated attributes
+         * @throws Exception error in the stock
+         */
+        public static IssuingStock.Log get(String id) throws Exception {
+            return IssuingStock.Log.get(id, null);
+        }
+
+        /**
+         * Retrieve a specific IssuingStock Log
+         * <p>
+         * Receive a single IssuingStock Log object previously created by the Stark Infra API by passing its id
+         * <p>
+         * Parameters:
+         * @param id [string]: object unique id. ex: "5656565656565656"
+         * @param user [Organization/Project object, default null]: Organization or Project object. Not necessary if starkinfra.Settings.user was set before function call
+         * <p>
+         * Return:
+         * @return IssuingStock Log object with updated attributes
+         * @throws Exception error in the stock
+         */
+        public static IssuingStock.Log get(String id, User user) throws Exception {
+            return Rest.getId(data, id, user);
+        }
+
+        /**
+         * Retrieve IssuingStock Logs
+         * <p>
+         * Receive a generator of IssuingStock.Log objects previously created in the Stark Infra API.
+         * Use this function instead of page if you want to stream the objects without worrying about cursors and pagination.
+         * <p>
+         * Parameters:
+         * @param params map of parameters for the query
+         * limit [integer, default null]: maximum number of objects to be retrieved. Unlimited if null. ex: 35
+         * after [string, default null] date filter for objects created only after specified date. ex: "2020-03-10"
+         * before [string, default null] date filter for objects created only before specified date. ex: "2020-03-10"
+         * types [list of strings, default null]: filter retrieved objects by types. ex: "created", "spent", "restocked", "lost"
```

Comment:
Só uma sugestão, nesse caso do parâmetro `types`, só são possíveis esse valores que estão no examplo? Se sim, não acha melhor colocar Options: (opções) ao invés de ex: (opções)?

Reply:
Outra coisa, se é uma list of strings, não deveria ser: `ex: ["created", "spent", "restocked", "lost"]` ?

Reply:
boa boa coloquei [] q nesse caso seria um exemplo mesmo